Assets/Scripts/Player.cs

```diff
@@ -49,9 +51,60 @@ public class Player : MonoBehaviour
 
 	private string openVideo;
 
-	void Start ()
+	private static class ConnectedDevices
 	{
-		XRSettings.enabled = false;
+		public enum LoadedDevice
+		{
+			None,
+			Oculus,
+			OpenVr
+		}
+
+		public static LoadedDevice loadedDevice;
+
+		public static bool hasLeftController { get; set; }
+		public static bool hasRightController { get; set; }
+		public static bool hasRemote { get; set; }
+
+		public static bool HasDevice()
+		{
+			return hasLeftController || hasRightController || hasRemote;
+		}
+
+		public static void DetectDevices()
+		{
+			var devices = Input.GetJoystickNames();
+
+			switch (loadedDevice)
+			{
+				case LoadedDevice.Oculus:
+					hasLeftController = devices.Contains("Oculus Touch Controller - Left");
```

Comment:
These magic strings are fine for now, but if you ever need to use them in multiple places, put them in a const variable somewhere.